src/services/rebrandly/rebrandly.service.ts

```diff
@@ -0,0 +1,40 @@
+import { er } from "../services";
+import { EventsRepository } from "hackoss";
+import { PlatformService } from "../platform/platform.service";
+import * as Rebrandly from "rebrandly";
+import { RebrandlyLink } from "./link.rebrandly";
+import * as _ from 'lodash';
+export class RebrandlyService extends PlatformService{
```

Comment:
Where do you intend to place it then? Because it's going to be triggered on publish, if it's within EB or FB, it would be called twice(once per platform)

Reply:
Agreed, it shouldn't be there in the publish functions as well as it will violate the [SRP](https://www.toptal.com/software/single-responsibility-principle). I suppose it should be called from the admin page (client), but our job here in the API is to just make it easier there to do so.